### PR TITLE
use eslint-config-standard

### PR DESCRIPTION
**Pull request by KaelWD**
## Description
I think this should be good, but I had to make some logic changes to appease eslint. Please triple-check all of these as some of them are rather convoluted. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: src/components/VDataTable/mixins/head.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default {
         }
       }
 
-      if ('sortable' in header && header.sortable || !('sortable' in header)) {
+      if (header.sortable === true || header.sortable == null) {
```
Comment (KaelWD) [inline src/components/VDataTable/mixins/head.js]: `header.sortable || header.sortable == null`

File: src/components/VDialog/VDialog.js [hunk 2]
```diff
@@ -171,9 +171,10 @@ export default {
     }, [dialog]))
 
     return h('div', {
-      'class': 'dialog__container',
+      staticClass: 'dialog__container',
       style: {
-        display: !this.$slots.activator && 'none' || this.fullWidth ? 'block' : 'inline-block'
+        // TODO: double-check this logic, it might need 'none'
+        display: (!this.$slots.activator || this.fullWidth) ? 'block' : 'inline-block'
```
Comment (KaelWD) [inline src/components/VDialog/VDialog.js]: 99c586cb056b71188d8b558426e34554c7b4d348

File: src/mixins/input.js [hunk 3]
```diff
@@ -85,10 +85,8 @@ export default {
     genMessages () {
       let messages = null
 
-      if ((this.hint &&
-            this.isFocused ||
-            this.hint &&
-            this.persistentHint) &&
+      if (((this.hint && this.isFocused) ||
+          (this.hint && this.persistentHint)) &&
```
Comment (KaelWD) [inline src/mixins/input.js]: `this.hint && (this.isFocused || this.persistentHint)`

File: src/components/VDialog/VDialog.js (hunk 2 above)
Comment (johnleider) [reply src/components/VDialog/VDialog.js]: reverted to original logic

**Pull request merged**